agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of the Agent
-const Version = "1.86.0"
+const Version = "1.86.1"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "7ca8b579"
+const GitShortHash = "fd8cbeff"
```

Comment:
this hash would need to be updated. Now with [this](https://github.com/aws/amazon-ecs-agent/commit/c9c748276a1e8ddf2bd56d1f3efd07da34ab83e5) merged and the tip of dev branch, it should to be `c9c74827`

Reply:
Resolved in https://github.com/aws/amazon-ecs-agent/commit/36fda28eb0fc51266890590ea781ff733fb5e688